migrations/migrations/20211102135840-user.js

```diff
@@ -1,4 +1,5 @@
 'use strict';
+const { ROLE } = require('../../config/constants/users_constants');
```

Comment:
Do not set dependencies in migrations. Migrations are preferred to be literal because the dependency can eventually change and the migration must be able to be executed from scratch at any time.